tests/vcr_python_wrapper.py

```diff
@@ -121,9 +146,24 @@ def filter_request_manifest(request):
     return request
 
 
+def filter_request_ldap(request):
+    request.uri = urlunparse(urlparse(request.uri)._replace(netloc="foreman.example.org"))
+    if request.body is not None:
+        try:
+            json_body = json.loads(request.body)
+        except json.JSONDecodeError:
+            pass
+        else:
+            if 'auth_source_ldap' in json_body:
+                json_body['auth_source_ldap'] = 'FILTERED'
+                request.body = json.dumps(json_body).encode()
+    return request
+
+
 def filter_request(request):
     request = filter_request_uri(request)
     request = filter_request_manifest(request)
+    request = filter_request_ldap(request)
```

Comment:
:+1:

Reply:
This function seems to break a lot of existing tests. Maybe we need to restrict it to the user role test?

Reply:
Not sure how to do it, there are still some areas i don´t know/understand.

Reply:
we have some test specific filters like:
https://github.com/theforeman/foreman-ansible-modules/blob/develop/tests/vcr_python_wrapper.py#L161

But that variable is not available in the scope of `filter_request`.